samples/Templates/Scenarios/Restaurant.data.json

```diff
@@ -25,5 +25,5 @@
 		"description": "Great concept and a wide selection of beers both on tap and bottled! Smaller wine selection than I wanted, but the variety of beers certainly made up for that. Although I didn't order anything, my boyfriend got a beer and he loved it. Their prices are fair too. \n\nThe concept is really awesome. It's a bar/store that you can bring outside food into. The place was pretty packed tonight. I wish we had stayed for more than one drink. I would have loved to sample everything!",
 		"author": "Blaire S."
 	}],
-	"priceRange": "$$"
-}
\ No newline at end of file
+	"priceRange": "mid-priced"
```

Comment:
I wonder how they make google maps accessible, I've seen the $ or $$$ model work before

Reply:
Well, we would need to add some mechanism for authors to specify alternative text.

Could potentially be solved if/when we figure out tooltips on all elements? Not sure.

Reply:
Yeah, i agree. We should probably have something tracking figuring out alt-text on Text elements for this kind of scenario. 

---
In reply to: [485986018](https://github.com/microsoft/AdaptiveCards/pull/4762#discussion_r485986018) [](ancestors = 485986018)

Reply:
filed #4764 to track